index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>freeCodCamp Survey Form</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+<br>
+<form action="" id="Survey-from"></form>
+<label for="name">Name</label>
+<input type="text" name="name" id="name" placeholder="Enter your Name" required>
+
+<br>
+
+<label for="email">E-mail</label>
+<input type="email" name="email" id="email" placeholder="*******@gmail.com" required>
+
+<br>
+
+<label for="age">Age</label>
+<input type="age" name="age" id="age" placeholder="Enter your age" required>
+
+<br>
+
+<p>Which option best describes your current role?</p>
+<select name="role" id="drop down">
+    <option disabled selected value> Select current role</option>
+    <option value="student">Student</option>
+    <option value="Fulltime job">Fulltime job</option>
+    <option value="Fulltime learner">Fulltime learner</option>
+    <option value="Prefer not to say">Prefer not to say</option>
+    <option value="Other">Other</option>
+</select>
+<br>
+<p>Would you recommend freeCodeCamp to a firend</p>
+<input name = "user-recommend" value="Definitly" type="radio" checked>
+<input name = "user-recommend" value="Maybe" type="radio">
+<input name = "user-recommend" value="Not sure" type="radio">
+
+<br>
+<p>What is your favorite feature of freeCodeCamp</p>

```

Comment:
[OPTIONAL] I suggest replacing the `<p>` tag with `<label for="most-like">` 🤏
> By doing it would be better for the UX when the user clicks on this label, it will automatically focus on the select options
```html
<label for="most-like">What is your favorite feature of freeCodeCamp</label>
```